src/govuk/components/tag/template.test.js

```diff
@@ -75,4 +75,94 @@ describe('Tag', () => {
     expect($component.attr('data-test')).toEqual('attribute')
     expect($component.attr('id')).toEqual('my-tag')
   })
+
+  it('grey example passes accessibility tests', async () => {
```

Comment:
The `render` function just returns the HTML from the macro itself, without the layout that brings in the stylesheet. Unfortunately this means there's little benefit to running this for every example, as it won't be checking the contrast between the colours that will actually be used.

Reply:
We've talked about running axe on the review application itself in the future, which would achieve what you're looking to do.

Reply:
Shall I remove all of the tests I added @36degrees ?

Reply:
I think so – unfortunately we don't have a good way to test this sort of thing right now.